omnibus-arbitrum-sepolia.toml

```diff
@@ -14,7 +14,6 @@ include = [
     # USDC
     "tomls/oracles/pyth-usdc.toml",
     "tomls/omnibus-arbitrum-sepolia/collaterals/usdc.toml",
-    "tomls/omnibus-arbitrum-sepolia/rewards/ARB_rewards_for_USDC_lp.toml",
```

Comment:
I think it is ok to keep these files with one invoke on them. We will have a good git diff showing how one rd replaced by another one instead of completely changing the structure. 
but im not too concerned with this